### ISSUE TITLE
FATAL ERROR: uninitialized constant App::AppError::Ico

### ISSUE THREAD

**Issue by shanko** [waxxio/waxx#1]
I installed the waxx gem and started the demo app as per the instructions (Mac OS, ruby 2.6.3)

$ ruby --version
ruby 2.6.3p62 (2019-04-16 revision 67580) [x86_64-darwin18] . 

I see the error: "FATAL ERROR: uninitialized constant App::AppError::Ico" multiple times in the log. Also it appears that there other failures in loading the required CSS/JS. (screen shot attached).

<img width="1437" alt="Screen Shot 2019-08-07 at 11 15 46 PM" src="https://user-images.githubusercontent.com/10062/62674738-5d53fa00-b969-11e9-97fc-36f5c31771ca.png">

What am I missing?

**Comment by dfitzpat** [comment]
Thanks for trying Waxx Shanko. The gem installation does not include a favicon.ico in the public folder so if you add one, that error will go away. There are links to some normal js and css files. 

See `app/html.rb`. In this file you define your site-wide html. The following two functions specify what CSS and JS you want in your pages. Edit these to suit your needs. The paths are relative to the `public` folder in your Waxx installation location. So the `"/lib/bootstrap/css/bootstrap.min.css"` on line 7 would load Bootstrap from `public/lib/bootstrap/css/bootstrap.min.css`.

```
  5   def standard_css
  6     [
  7       "/lib/bootstrap/css/bootstrap.min.css",
  8       "/lib/bootstrap/css/carousel.css",
  9     ]
 10   end
 11
 12   def standard_js
 13     [
 14       "/lib/jquery-3.1.0.min.js",
 15       "/lib/bootstrap/js/bootstrap.min.js",
 16       "/app/js.js",
 17     ]
 18   end
```

I will update the default installation on the next release. I plan to have it show a better start page.  I'll leave this issue open until the next version has a fix.

Any other feedback you have is greatly appreciated.

**Comment by shanko** [comment]
So here is my modified **app/html.rb** snippet:

def standard_css
  [
    "/lib/bootstrap/css/bootstrap.min.css",
  ]
end

def standard_js
  [
    "/lib/bootstrap/js/jquery-3.3.1.slim.min.js",
    "/lib/bootstrap/js/bootstrap.min.js",
    "/lib/bootstrap/js/popper.min.js",
  ]
end

Here are the files in demo-site/lib:

~/demo-site/lib (master)
$ ls -ltr bootstrap/css
total 312
-rw-r--r--  1 shanko  wheel  155758 Feb 13 10:40 bootstrap.min.css
drwxr-xr-x  4 shanko  wheel     128 Aug  9 17:20 ..
drwxr-xr-x  3 shanko  wheel      96 Aug  9 17:33 .

~/demo-site/lib (master)
$ ltr bootstrap/js
total 312
-rw-r--r--  1 shanko  wheel  69917 Jan 20  2018 jquery-3.3.1.slim.min.js
-rw-r--r--  1 shanko  wheel  21004 Jan 29  2019 popper.min.js
-rw-r--r--  1 shanko  wheel  58072 Feb 13 10:40 bootstrap.min.js
drwxr-xr-x  4 shanko  wheel    128 Aug  9 17:20 ..
drwxr-xr-x  5 shanko  wheel    160 Aug  9 17:34 .

~/demo-site (master)
$ ltr public
total 0
drwxr-xr-x   4 shanko  wheel  128 Aug  3 15:00 lib
-rw-r--r--   1 shanko  wheel    0 Aug  9 17:11 favicon.ico
drwxr-xr-x   4 shanko  wheel  128 Aug  9 17:11 .
drwxr-xr-x  16 shanko  wheel  512 Aug  9 17:34 ..

Here are the errors I get:

$ grep FATAL log/waxx.log
FATAL ERROR: uninitialized constant App::AppError::Css
FATAL ERROR: uninitialized constant App::AppError::Js
FATAL ERROR: uninitialized constant App::AppError::Js
FATAL ERROR: uninitialized constant App::AppError::Js
FATAL ERROR: uninitialized constant App::AppError::Ico

<img width="1437" alt="Screen Shot 2019-08-09 at 5 49 25 PM" src="https://user-images.githubusercontent.com/10062/62813078-8f32a100-bace-11e9-9b18-a0374277ee0e.png">

**Comment by shanko** [comment]
Never mind .. I got it all working as expected. 

I was confused between <root>/lib and <root>/public/lib. Once I moved all CSS and JS in **<root>/public/lib** it all started working. 

Looking forward to the update  of the default installation and a better start page. 

Thanks!

**Comment by dfitzpat** [comment]
This issue is fixed in version 0.2.0.
